qiskit_nature/algorithms/pes_samplers/bopes_sampler.py

```diff
@@ -41,17 +42,20 @@
 class BOPESSampler:
     """Class to evaluate the Born-Oppenheimer Potential Energy Surface (BOPES)."""
 
+    @deprecate_arguments({"gss": "solver_wrapper"})
+    # pylint: disable=unused-argument
     def __init__(
         self,
-        gss: Union[GroundStateSolver, ExcitedStatesSolver],
+        solver_wrapper: Optional[Union[GroundStateSolver, ExcitedStatesSolver]] = None,
```

Comment:
From a user perspective do you think `solver_wrapper` is a good name? Since its either  a GSS or ESS why not just `solver` - since its one of GroundState or ExcitedStates solvers that a user is passing in. Yes internally the code might want to get at the raw algorithm solver but the "wrapper" aspect seems awkward to me looking at this purely from an end user using the algo

Reply:
Indeed we can change it to `solver`. Because we exposed the solver of ExcitedStatesEigensolver, `wrapper_solver` now does not feel necessary. Before that, it was to avoid using solver.solver.solve()

Reply:
When trying to access the `initial_point` of the solver we would use `self._solver.solver.initial_point`. I find it confusing so I went for` _state_solver`.